src/EnergyPlus/WindowAC.cc

```diff
@@ -1499,6 +1499,36 @@ namespace WindowAC {
         } // WindAC(WindACNum)%DXCoilType_Num == CoilDX_CoolingHXAssisted && *
     }
 
+    bool GetWindowACNodeNumber(EnergyPlusData &state, int const NodeNumber)
+    {
+        if (state.dataWindowAC->GetWindowACInputFlag) {
+            GetWindowAC(state);
+            state.dataWindowAC->GetWindowACInputFlag = false;
+        }
+
+        bool windowACOutdoorAir = false;
+
+        for (int windowACIndex = 1; windowACIndex <= state.dataWindowAC->NumWindAC; ++windowACIndex) {
+            auto &windowAC = state.dataWindowAC->WindAC(windowACIndex);
+            if (windowAC.OutAirVolFlow == 0) {
+                windowACOutdoorAir = true;
+            } else {
+                windowACOutdoorAir = false;
+            }
```

Comment:
Only allow `ZoneHVAC:WindowAirConditioner`s that do not provide outdoor air to not interfere with the AFN.